<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskDialog</class>
 <widget class="QDialog" name="TaskDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>232</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Editing task data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="layout_name">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_name">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Task name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_name"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layout_group">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_group">
         <property name="text">
          <string>Group</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_group"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layout_creation_date">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_creation_date">
         <property name="text">
          <string>Creation date</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="edit_creation_date"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkbox_complete">
       <property name="text">
        <string>Is completed?</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layout_completion_date">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_complete_date">
         <property name="text">
          <string>Completion date</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="edit_complete_date">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layout_dialog">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="button_ok">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_cancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
